packages/docusaurus-1.x/lib/core/BlogPageLayout.js

```diff
@@ -10,6 +10,8 @@ const BlogPost = require('./BlogPost.js');
 const BlogSidebar = require('./BlogSidebar.js');
 const Container = require('./Container.js');
 const MetadataBlog = require('./MetadataBlog.js');
+
+const MetadataPublicBlog = MetadataBlog.filter(item => item.unlisted !== true);
```

Comment:
Where would be a good place to put this shared logic between `BlogSidebar.js`, `BlogPageLayout.js` and `feed.js`?